src/function/cast_string_non_nested_functions.cpp

```diff
@@ -165,11 +165,29 @@ static RE2& realPattern() {
     return retval;
 }
 
+bool isAny(std::string_view cpy) {
+    std::string cpy_upper = std::string(cpy);
```

Comment:
I would suggest renaming to `isAnyType`?

Reply:
renamed